lambda_function/lambda

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
This file is no tf file and is in a subfolder

Reply:
done